src/ResourceManager/DataMigration/AzureRM.DataMigration.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '0.2.0'
```

Comment:
@hitenjava same comment -- we will handle versioning the module

Reply:
Done